### ISSUE TITLE
GridFS support

### ISSUE THREAD

**Issue by slyons** [countercept/mongo-rs#5]
Is there any chance that you'd be up for supporting GridFS at any time in the future?

**Comment by alexkornitzer** [comment]
Yep, the goal is to support all the features assuming it makes sense on this abstraction layer. I have never used GridFS properly though so it will require some digging. I'll see if I can carve out some time to have a play with this.

**Comment by alexkornitzer** [comment]
Hmm, looks like the officials have not implemented GridFS support yet https://github.com/mongodb/mongo-rust-driver and this is only a wrapper crate. No promises, but we might be able to bodge a solution on top working from their labs implementation long ago. (https://docs.rs/mongodb/0.2.9/mongodb/gridfs/index.html)
